lib/modules/dosomething/dosomething_reportback/includes/dosomething_reportback.inc

```diff
@@ -333,7 +333,7 @@ class ReportbackEntityController extends EntityAPIController {
     }
 
     parent::save($entity, $transaction);
-
+    
```

Comment:
bah! extra spaces! Install [editor config](https://github.com/sindresorhus/atom-editorconfig) to avoid this stuffs